lib/phoenix_live_view/utils.ex

```diff
@@ -238,17 +238,23 @@ defmodule Phoenix.LiveView.Utils do
   Returns the configured signing salt for the endpoint.
   """
   def salt!(endpoint) when is_atom(endpoint) do
-    endpoint.config(:live_view)[:signing_salt] ||
+    salt = endpoint.config(:live_view)[:signing_salt]
+
+    if is_binary(salt) and byte_size(salt) >= 16 do
```

Comment:
I took the value `16` from the `byte_size` of the output of `Phoenix.LiveView.Utils.random_encoded_bytes/0`

Reply:
The salt does not need to be that strong because the secret key base is really the one that matters. So I will reduce this to 8 and we should be good. :)

Reply:
```suggestion
    if is_binary(salt) and byte_size(salt) >= 8 do
```